### ISSUE TITLE
Request: one-liner summary after changing files

### ISSUE THREAD

**Issue by irontoby** [wp-cli/wp-cli#1934]
`wp-cli` + `git` is WordPress admin nirvana for me! Thanks so much for turbo charging my workflow.

I'd love to get a one-line summary of what changed when running various install/update commands, so I can easily grab it as my git commit message. Something like __Installed "zerif-lite" theme 1.8.2.2__ or __Updated "akismet" plugin 3.1.1 -> 3.1.3__ ... of course for commands which perform batch updates/installs this would be multiple lines. (I'm thinking the message could go right above the existing __Success: Updated 1/1 plugins__)

Is this something you'd consider useful? If so I could look at hacking at it myself. Thanks again for a wonderful tool!

**Comment by danielbachhuber** [comment]
I took a little look into this. What would be ideal would be to change:

```
Downloading update from https://downloads.wordpress.org/plugin/wp-gistpen.0.5.7.zip...
Unpacking the update...
Installing the latest version...
Removing the old version of the plugin...
Plugin updated successfully.
```

to 

```
Downloading update from https://downloads.wordpress.org/plugin/wp-gistpen.0.5.7.zip...
Unpacking the update...
Installing the latest version...
Removing the old version of the plugin...
Gistpen updated successfully from version 0.5.5 to version 0.5.7.
```

However, it doesn't appear we have easy access to that change because `WP_CLI\UpgraderSkin` know of the scope in which it's being run. 

In #1358, we added a table summary like this:

```
Success: Updated 4/4 plugins.
+------------------+-------------+-------------+---------+
| name             | old_version | new_version | status  |
+------------------+-------------+-------------+---------+
| co-authors-plus  | 2.6.1       | 3.1.1       | Updated |
| demo-plugin      | 0.1-alpha   | 1.0         | Updated |
| shortcode-ui/dev | v1.0        | 0.4.0       | Updated |
| wp-gistpen       | 0.4.0       | 0.5.7       | Updated |
+------------------+-------------+-------------+---------+
```

We could add a new `--format=<format>` argument, default to `table` and also support `summary` as a one-liner format.

**Comment by danielbachhuber** [comment]
#1936 produces output like this:

```
salty-wordpress ➜  wordpress-develop.dev  wp plugin update --all --format=summary
Enabling Maintenance mode...
Downloading update from https://downloads.wordpress.org/plugin/co-authors-plus.3.1.1.zip...
Using cached file '/home/vagrant/.wp-cli/cache/plugin/co-authors-plus-3.1.1.zip'...
Unpacking the update...
Installing the latest version...
Removing the old version of the plugin...
Plugin updated successfully.
Disabling Maintenance mode...
Success: Translations updates are not needed for the 'English (US)' locale.
Success: Updated 1/1 plugins.
Co-Authors Plus updated successfully from version 2.6.1 to version 3.1.1
```

However, I think we should just suppress everything but the single lines.

**Comment by irontoby** [comment]
Thanks for looking into it! So it looks like this functionality would only be for `update --all` and not updating individual items (e.g. `wp-cli plugin update akismet`)?

**Comment by danielbachhuber** [comment]
> So it looks like this functionality would only be for update --all and not updating individual items (e.g. wp-cli plugin update akismet)?

Incorrect. It would also be available for the latter.

**Comment by irontoby** [comment]
Well that's exactly what I needed then, thanks!! At that point I could even write a script to parse the result & automatically commit changes on success (I prefer having each theme/plugin install/update as a separate commit in case I need to rollback later...)

**Comment by danielbachhuber** [comment]
`--format=summary` will be available in v0.20.0
